_posts/2023-03-28-cflinuxfs3-deprecation.md

```diff
@@ -93,4 +93,4 @@ cf curl "/v3/apps?per_page=5000&include=space.organization" | jq '(.included.spa
 
 ### Update
 
-The original instructions indicated that a `cf restage` will move apps to cflinuxfs4.  It does not, you have to `cf push` your application.
\ No newline at end of file
+The original instructions indicated that a `cf restage` will move apps to cflinuxfs4.  It does not, you have to `cf push -s STACK_NAME` your application.
```

Comment:
I would have put this update at the top (below the heading) in bold, with a date of when the update was made for context, but It's not necessary at this stage

Reply:
Made the change and pushed another commit, thanks for the suggestion